apache2/mod_security2.c

```diff
@@ -95,9 +95,11 @@ TreeRoot DSOLOCAL *conn_write_state_suspicious_list = 0;
 #ifdef WAF_JSON_LOGGING_ENABLE
 char DSOLOCAL *msc_waf_resourceId = "";
 char DSOLOCAL *msc_waf_instanceId = "";
-char DSOLOCAL *msc_waf_lock_owner = "root";
 #endif
 
+char DSOLOCAL *msc_waf_lock_user;
+char DSOLOCAL *msc_waf_lock_group;
```

Comment:
We should use NULL to initialize msc_waf_lock_user and msc_waf_lock_group although you initialize them in ap_hook_set_lock_owner(hook_set_lock_owner, NULL, NULL, APR_HOOK_MIDDLE)

Reply:
fix